docs/developers/surf-testmachine-setup.md

```diff
@@ -17,9 +17,9 @@ Before beginning:
 7. Install Surf:
     - The quickest way to install Surf is with NPM:
     `npm install -g surf-build`
-    - OR you can download a release from https://github.com/surf-build/surf/releases.
-8. Configure the windows machine to [automatically log in on boot](https://www.howtogeek.com/112919/how-to-make-your-windows-8-computer-logon-automatically/). 
-9. In an *administrative cmd shell*, 
+    - Alternatively, download a release from https://github.com/surf-build/surf/releases.
```

Comment:
This isn't user-oriented documentation (in docs/developers. I'm not sure there's a lot of value in working on it.  But does no harm.

Reply:
Agreed that this is more for our testing strategy and not necessarily intended for a typical ddev user unless they are attempting to write tests against PRs. That said, this change is ok by me.

Reply:
ok!